dotcom-rendering/src/web/components/ArticleContainer.tsx

```diff
@@ -96,6 +96,7 @@ const adStyles = css`
 
 		/* liveblogs ads have different background colours due the darker page background */
 		.ad-slot--liveblog-inline {
+			min-height: 274px;
```

Comment:
It isn't immediately obvious where this value of 274px comes through, is it 250px for the ad plus the height of the label? If so, maybe we can do something similar to [here](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/components/HeaderAdSlot.tsx#L18)

```suggestion
			min-height: ${250 + labelHeight}px;
```

Reply:
Thanks, I have amended

Reply:
Thanks!